gravitee-am-reporter/gravitee-am-reporter-kafka/src/test/resources/json/schema.json

```diff
@@ -0,0 +1 @@
+{"$schema":"http://json-schema.org/draft-07/schema#","title":"Audit Message Value Dto","type":"object","additionalProperties":false,"properties":{"id":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"transactionId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"type":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"referenceType":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"referenceId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"accessPoint":{"oneOf":[{"type":"null","title":"Not included"},{"$ref":"#/definitions/AuditAccessPointDto"}]},"actor":{"oneOf":[{"type":"null","title":"Not included"},{"$ref":"#/definitions/AuditEntityDto"}]},"target":{"oneOf":[{"type":"null","title":"Not included"},{"$ref":"#/definitions/AuditEntityDto"}]},"status":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"timestamp":{"oneOf":[{"type":"null","title":"Not included"},{"type":"integer","format":"utc-millisec"}]},"environmentId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"organizationId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"domainId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"nodeId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"nodeHostname":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]}},"definitions":{"AuditAccessPointDto":{"type":"object","additionalProperties":false,"properties":{"id":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"alternativeId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"displayName":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"ipAddress":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"userAgent":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]}}},"AuditEntityDto":{"type":"object","additionalProperties":false,"properties":{"id":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"alternativeId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"type":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"displayName":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"referenceType":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]},"referenceId":{"oneOf":[{"type":"null","title":"Not included"},{"type":"string"}]}}}}}
```

Comment:
**question**: is there any reason why you didn't provide the schema in petty format?

Reply:
This is how schema registry store schemas.

Reply:
does it means we can't have the "formatted" version in the repo because the registry will not flatten the content before checking if the schema is already known?